docs/sources/flow/tutorials/flow-by-example/faq.md

```diff
@@ -0,0 +1,35 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tutorials/flow-by-example/faq/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/tutorials/flow-by-example/faq/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/tutorials/flow-by-example/faq/
+- /docs/grafana-cloud/send-data/agent/flow/tutorials/flow-by-example/faq/
+canonical: https://grafana.com/docs/agent/latest/flow/tutorials/flow-by-example/faq/
+description: FAQ About Flow-by-Example Tutorials
+title: FAQ
```

Comment:
```suggestion
title: Get started
```

Could we title this another way? FAQ is... something I'd like to avoid if we can.